hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -735,6 +735,11 @@ protected void addMapEntrySet(Map<Object, Object> entries, MapEntries mapEntries
     }
 
     @Override
+    public boolean existInMemory(Data key) {
+        return storage.containsKey(key);
```

Comment:
@ahmetmircik Good catch with this checkIfLoaded, I was also in doubt that it is needed here ;)